spec/models/group_order_article_spec.rb

```diff
@@ -55,9 +64,13 @@
 
     it 'can calculate the result for the distribution strategy "no automatic distribution"' do
       FoodsoftConfig[:distribution_strategy] = FoodsoftConfig::DistributionStrategy::NO_AUTOMATIC_DISTRIBUTION
-
```

Comment:
please try to avoid unrelated code changes, if they are not done by a automatic code formatting tool